### PR TITLE
all: use loggo rather than log for logging

### PR DESCRIPTION
**Pull request by rogpeppe**
Also update the dependencies.tsv file, which hasn't been updated for a while.

### CODE REVIEW

**Comment by jrwren** [comment]
:+1: :man:

File: bakery/storage.go [hunk 1]
```diff
@@ -52,10 +52,10 @@ func (s *memStorage) Get(location string) (string, error) {
 	defer s.mu.Unlock()
 	item, ok := s.values[location]
 	if !ok {
-		logf("storage.Get[%q] -> not found", location)
+		logger.Infof("storage.Get[%q] -> not found", location)
```
Comment (mhilton) [inline bakery/storage.go:55]: Should this possibly be higher than Info Error maybe?

File: httpbakery/client.go [hunk 2]
```diff
@@ -147,9 +149,9 @@ func relativeURL(base, new string) (*url.URL, error) {
 }
 
 func (ctxt *clientContext) do(req *http.Request, getBody BodyGetter) (*http.Response, error) {
-	log.Printf("client do %s %s {", req.Method, req.URL)
+	logger.Debugf("client do %s %s {", req.Method, req.URL)
 	resp, err := ctxt.do1(req, getBody)
-	log.Printf("} -> error %#v", err)
+	logger.Infof("} -> error %#v", err)
```
Comment (mhilton) [inline httpbakery/client.go]: Might this look a little odd if logging at info level or above, the curly brackets won't match.

File: httpbakery/client.go [hunk 3]
```diff
@@ -271,7 +273,7 @@ func (ctxt *clientContext) obtainThirdPartyDischarge(originalLocation string, ca
 	if err == nil {
 		return resp.Macaroon, nil
 	}
-	log.Printf("discharge post got error %#v", err)
+	logger.Debugf("discharge post got error %#v", err)
```
Comment (mhilton) [inline httpbakery/client.go]: Is this really a debug level message?

**Comment by mhilton** [comment]
:+1: LGTM, but check you're happy with the log levels.

File: httpbakery/client.go (hunk 2 above)
Comment (rogpeppe) [inline httpbakery/client.go]: Ha, they should both be at debug level.

File: httpbakery/client.go (hunk 3 above)
Comment (rogpeppe) [inline httpbakery/client.go]: We should probably just delete the log message - the error will be returned and should be logged anyway by our caller.

File: bakery/storage.go (hunk 1 above)
Comment (rogpeppe) [inline bakery/storage.go:55]: Not sure. The error should be logged elsewhere anyway. It seems to me like the same level as the other storage access log messages is appropriate.

**Pull request merged**